spec/vcr_cassettes/manageiq/providers/openshift/container_manager/refresher_after_openshift_deletions.txt

```diff
@@ -0,0 +1,39 @@
+VCR generated by spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh against 192.168.99.100
```

Comment:
File name isn't clear to me as it's openshift objects and not refresh or manageiq related

Reply:
You mean the .txt name, or test_objects_record.sh name?
.txt name is just mirroring .yml name.
All names half-thought, better names welcome :)

I wonder if script should go to tools/ instead of deep under spec/?

Reply:
.txt name
suggestion `/refresher_after_openshift_deletions.txt/post_deletion_objects.txt/`

> I wonder if script should go to tools/ instead of deep under spec/?

Will this need distribution e.g would we be interested in run the script not from a manageiq copy
(in some automation flow)?